test/metrics/metrics.jl

```diff
@@ -0,0 +1,215 @@
+using ADRIA: metrics
+using ADRIA: DataFrame, DataFrameRow, names
+using ADRIA: YAXArray, DataCube, ZeroDataCube
+
+if !@isdefined(TEST_RS)
+    const TEST_DOM, TEST_N_SAMPLES, TEST_SCENS, TEST_RS = test_rs()
+end
+
+function _test_metrics(
+    metric::metrics.Metric, params::Tuple; relative::Bool=false
+)::Nothing
+    metric_result = metric(params...)
+    @test all(haskey.([metric_result.properties], (:metric_name, :metric_unit)))
+    @test all(0.0 .<= metric_result.data)
+    relative && @test all(metric_result.data .<= 1.0)
+    return nothing
+end
+
+function test_relative_metric(metric::metrics.Metric, params::Tuple)::Nothing
+    return _test_metrics(metric, params; relative=true)
+end
+
+function test_absolute_metric(metric::metrics.Metric, params::Tuple)::Nothing
+    return _test_metrics(metric, params; relative=false)
+end
```

Comment:
This could be simplified by checking if `absolute` or `relative` appears in the metric name.

I believe all coral cover metrics should follow that naming pattern.

Then you don't need the specific `test_relative_*`/`test_absolute_*` functions.